inst/rmarkdown/templates/problemset/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,140 @@
+---
+####################
+## Metadata       ##
+####################
+course: Mathematics of Ring-bearing
+title: |
+  Problem Set 1
+code: MINT001 # Course code for filename.
+date: "`r paste(Sys.Date())`" # Inserts today's date
+authors: Gandalf the Grey, Frodo Baggins
+# This is the YAML (YAML Ain't Markup Language) header that includes 
+# metadata used to produce the document. 
+# Be careful with spacing in this header!
+# Two spaces at the start of the line create indentation,
+# which embeds the parameter in a hierarchy.
+
+#####################
+## Bibliography    ##
+#####################
+bibliography: [references.bib, packages.bib]
+csl: chicago-author-date.csl
+nocite: '@*' # Adds all elements in bib file to bibliography
+
+##########################
+## Define Knit function ##
+##########################
+knit: iheiddown::problemset_pdf
+
+#####################
+## Output Options  ##
+#####################
+output: iheiddown::problemset_pdf
```

Comment:
There seems to be an error when I try and run this:

```
! Package fontspec Error: The font "Latin Modern Roman" cannot be found.

! kpathsea:make_tex: Invalid filename `Latin Modern Roman', contains ' '

Error: LaTeX failed to compile MINT001_ProblemSet_2021-08-21.tex. See https://yihui.org/tinytex/r/#debugging for debugging tips. See MINT001_ProblemSet_2021-08-21.log for more info.
Execution halted
```

Reply:
Good spot! I have changed the fonts to be consistent with our thesis template and work via Latex packages rather than user system font imports. Could you confirm that it works now on macOS as well @jhollway ?